istioctl/cmd/istioctl/rbac.go

```diff
@@ -84,6 +87,35 @@ istioctl experimental rbac can -s source.namespace=foo POST rating /data -a dest
 	return cmd
 }
 
+func upgrade() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "upgrade <rbac-file>",
+		Short: "Upgrade Istio Authorization policy file to use the newest API",
+		Long: `
+This command automatically converts Istio Authorization v1 policy files to v2. It requires the operator 
+to have READ access to the Kubernetes cluster where the services specified in ServiceRole definitions exist. 
+`,
+		Args: cobra.MinimumNArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			upgrader, err := newUpgrader(args[0])
```

Comment:
Elsewhere in Istio we use `-f` for filename arguments.  Please follow that pattern or explain why you can't.

Reply:
Wasn't aware of this. Will look into it. Thanks